setup.py

```diff
@@ -1,31 +1,40 @@
+import os
+
 from setuptools import find_packages, setup
 
 from qdax import __version__
 
+CURRENT_DIR = os.path.abspath(os.path.dirname(__file__))
+
+with open(os.path.join(CURRENT_DIR, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
 setup(
     name="qdax",
     version=__version__,
     packages=find_packages(),
     url="https://github.com/adaptive-intelligent-robotics/QDax",
     license="MIT",
-    author="Bryan Lim and InstaDeep Ltd",
-    author_email="",
-    description="A Python Library for Quality Diversity and NeuroEvolution",
+    author="AIRL and InstaDeep Ltd",
+    author_email="adaptive.intelligent.robotics@gmail.com",
+    description="A Python Library for Quality-Diversity and NeuroEvolution",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
     install_requires=[
-        "absl-py==1.0.0",
-        "jax==0.3.2",
-        "flax==0.4.1",
-        "brax==0.0.12",
+        "absl-py>=1.0.0",
```

Comment:
Are you sure about those `>=`? As discussed recently (#26), it is safer to use `==` for the dependencies. 

Are the good practice different for the `install_requires`?

Reply:
I think the `install_requires` from `setup.py` are quite different from the `requirements.txt`.

Basically, I changed it following the guidelines of the Python Package Authority documentation: 
https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/

What do you think?

Reply:
Interesting! Thanks for the link! Indeed, this seems fine :slightly_smiling_face:

Reply:
Hi everyone, I'm passing through! I can confirm that `setup.py` should have `>=` -- try to use the lowest version possible that still allows your library to work, so that users have as much flexibility as possible. If anything, you may want to indicate it should be less than the next major version (e.g. `>=1.0.0,<2.0.0`) so that you're robust to API changes, but I don't think that's an issue here.